packages/evolution-frontend/src/styles/survey/_admin.scss

```
@@ -60,10 +33,92 @@
+        background: rgba(0, 0, 0, 0.025);
+        border-color: rgba(0, 0, 0, 0.12);
+    }
+
+    &._selectable:hover {
+        cursor: pointer;
+    }
+
+    &--has-review {
+        // Reserve top-right space for absolutely positioned review controls on this box.
+        $review-controls-width: 9.5rem;
+        padding-right: calc(0.5rem + #{$review-controls-width});
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)